Dockerfile.dapper

```diff
@@ -0,0 +1,61 @@
+FROM ubuntu:16.04
+
+ARG DAPPER_HOST_ARCH
+ENV HOST_ARCH=${DAPPER_HOST_ARCH} ARCH=${DAPPER_HOST_ARCH} DAPPER_ENV=REPO DAPPER_ENV=TAG \
+    DAPPER_SOURCE=/go/src/github.com/submariner-io/lighthouse DAPPER_DOCKER_SOCKET=true \
+    TRASH_CACHE=${DAPPER_SOURCE}/.trash-cache HOME=${DAPPER_SOURCE} DAPPER_OUTPUT=output
+
+RUN rm -f /bin/sh && ln -s /bin/bash /bin/sh
+
+ENV GOLANG_ARCH_amd64=amd64 GOLANG_ARCH_arm=armv6l GOLANG_ARCH=GOLANG_ARCH_${ARCH} \
+    GOPATH=/go GO111MODULE=on PATH=/go/bin:/usr/local/go/bin:${PATH} SHELL=/bin/bash GOFLAGS=-mod=vendor
+
+# Requirements:
+# Component       | Usage
+# -----------------------------------------------------------
+# gcc             | ginkgo
+# git             | find the workspace root
+# curl            | download other tools
+# docker.io       | Dapper
+# make            | coredns make
+# golang          | build
+# kubectl         | e2e tests
+# golangci-lint   | code linting
+# helm            | e2e tests
+# kubefedctl      | e2e tests
+# kind            | e2e tests
+# ginkgo          | tests
+# goimports       | code formatting
+
+RUN apt-get -q update && \
+    apt-get install -y gcc git curl docker.io make
+
+ENV GO_VERSION=1.12.6
```

Comment:
isn't it possible to always point to the latest release and to have another file to override versions if needed?

(same comment for all other _VERSION)